site/_collections/algolia.js

```diff
@@ -100,6 +100,22 @@ module.exports = collections => {
       enumerable: true,
     });
 
+    // Add API types to algolia doc to allow fully qualified searches,
+    // like chrome.webRequest.onBeforeSendHeaders instead of just being
+    // able to search onBeforeSendHeaders which also appears in the content
+    if (item.data.api && item.data.chromeApiNamespaces[item.data.api]) {
+      const apiNamespace = item.data.chromeApiNamespaces[item.data.api];
+
+      const apiNamespacePaths = [];
+      for (const group of apiNamespace.groups) {
+        for (const content of group.contents) {
+          apiNamespacePaths.push(content._name);
```

Comment:
_name underscore suggests a private member. Consider changing it in chrome-types?

Reply:
To avoid another loop over contents in group-types I now opted to build the path by using the available public members. WDYT?